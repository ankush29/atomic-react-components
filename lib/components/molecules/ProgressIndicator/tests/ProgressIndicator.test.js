import React from 'react';
import { shallow, render } from 'enzyme';

import { ProgressIndicatorVanilla } from '../index';

describe('<ProgressIndicator />', () => {
  let ProgressIndicatorComponent = '';
  beforeEach(() => {
    ProgressIndicatorComponent = shallow(<ProgressIndicatorVanilla>Test</ProgressIndicatorVanilla>);
  });

  test('should render correctly', () => {
    expect(ProgressIndicatorComponent).toMatchSnapshot();
  });
});
