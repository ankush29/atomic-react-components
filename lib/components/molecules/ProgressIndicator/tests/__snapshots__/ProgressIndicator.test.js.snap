// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProgressIndicator /> should render correctly 1`] = `
<div
  className="RSPBprogressBar"
  style={
    Object {
      "background": null,
      "height": null,
      "width": null,
    }
  }
>
  <Component
    accomplished={false}
    index={0}
    key=".0"
    position={NaN}
  />
  <div
    className="RSPBprogression"
    style={
      Object {
        "background": null,
        "width": "NaN%",
      }
    }
  />
</div>
`;
